tests/test_qnode.py

```diff
@@ -269,7 +269,7 @@ def circuit(x):
 

```

Comment:
Aside from the above changes, I didn't have to make any other changes. I looked at `test_quantum_gradients`, `test_classical_gradients`, and `test_qnode`, and all required trivial changes; either

- Setting `argnum` in the call to `grad`, or
- Settings `requires_grad=True`

However, more issues might come to light as more of the tests are explored